cmd/statusd/main.go

```diff
@@ -106,6 +108,12 @@ func main() {
 		go startCollectingStats(interruptCh, backend.NodeManager())
 	}
 
+	// Sync blockchain and stop.
+	if *syncAndExit >= 0 {
+		syncAndStopNode(backend.NodeManager())
```

Comment:
It's better to reduce the number of functions that do `os.Exit()`. So maybe we could have `syncAndStopNode` to simply return `error` if there was any and do `log.Fatalf()` here in one place.